### PR TITLE
Fixes yaml errors for b-bug-report.yml

### PR DESCRIPTION
**Pull request by JeroenWeener**
#1269 introduced a new bug report template. This template contained two issues that are resolved in this PR.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/b-bug-report.yml [hunk 1]
```diff
@@ -84,8 +85,6 @@ body:
       label: Version
       description: In which version of the geolocator plugin does the bug occur?
       placeholder: ex. 1.2.3
-      validations:
-        required: true
```
Comment (mvanbeusekom) [inline .github/ISSUE_TEMPLATE/b-bug-report.yml:88]: Does this mean we cannot force users to enter the version number of the plugin they use?

File: .github/ISSUE_TEMPLATE/b-bug-report.yml [hunk 2]
```diff
@@ -11,6 +11,7 @@ body:
         will allow us to quickly help you and resolve the issue you are facing.
   - type: checkboxes
     attributes:
+      label: Please check the following before submitting a new issue.
```
[the remark this answers is not in the document]
Comment (JeroenWeener) [reply .github/ISSUE_TEMPLATE/b-bug-report.yml:14]: Users cannot submit the issue without checking these boxes, as they are required. Would that be enough? Or do you insist on elaborating on the reasoning behind the tasks?

File: .github/ISSUE_TEMPLATE/b-bug-report.yml (hunk 1 above)
Comment (JeroenWeener) [reply .github/ISSUE_TEMPLATE/b-bug-report.yml:88]: Oops, turns out to be an indentation issue. I added it back in.

**Review by TimHoogstrate — APPROVED** [review]

**Review by mvanbeusekom — APPROVED** [review]
LGTM!

**Pull request merged**